### PR TITLE
Home Page Done

### PR DESCRIPTION
**Pull request by LikeInReal**
home page

### CODE REVIEW

File: SiteAsii/src/app/about/about.component.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h2>Despre noi...</h2>
   </div>
   <div class="question">
     <h3>Eşti curios din cine este alcătuită A.S.I.I. ?</h3>
-    <button mat-stroked-button class="view-button">Vezi membrii ASII</button>
+    <button mat-stroked-button class="view-button" routerLink="/membrii">Vezi membrii ASII</button>
```
Comment (teodragoi) [inline SiteAsii/src/app/about/about.component.html:59]: Remove the routerLink, it's not necessary for now

File: SiteAsii/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
-import { NgModule } from '@angular/core';
+import { NgModule, Component } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { HomeComponent } from 'src/app/home/home.component';
+import { MembriiComponent } from 'src/app/membrii/membrii.component';
 const routes: Routes = [
+  {
+    path: 'membrii', component: MembriiComponent,
+  },
```
Comment (teodragoi) [inline SiteAsii/src/app/app-routing.module.ts:7]: Also remove this

File: SiteAsii/src/app/home/home.component.scss [hunk 3]
```diff
@@ -14,26 +14,19 @@
...
+        color: #E50D0E;
+        font-size: 1.6rem;
+        font: Arial;
+        margin-bottom: 1.5em;
     }
-    h3 {
-        color: white;
-        font-size: 1rem;
-        margin-top: 3px;
+    .MOTTO {
```
Comment (teodragoi) [inline SiteAsii/src/app/home/home.component.scss:26]: Don't use upper case in class names

File: SiteAsii/src/app/about/about.component.html (hunk 1 above)
Comment (LikeInReal) [reply SiteAsii/src/app/about/about.component.html:59]: Asta nu e facuta de mine

**Pull request merged**